Rakefile

```diff
@@ -19,9 +19,9 @@ def next_version
 end
 
 task :tag do
-  sh "docker tag -f parabuzzle/craneoperator:latest parabuzzle/craneoperator:#{next_version}"
-  sh "docker tag -f parabuzzle/craneoperator:latest parabuzzle/docker-registry-ui:latest"
-  sh "docker tag -f parabuzzle/craneoperator:latest parabuzzle/docker-registry-ui:#{next_version}"
+  sh "docker tag parabuzzle/craneoperator:latest parabuzzle/craneoperator:#{next_version}"
```

Comment:
the "-f" is on all these tag calls because of a problem with the way circleci works and a minor optimization I've made in the build time on that platform. Please put these back so that the automation doesn't break.